sydneywait/AppalachianHarvest
# User can register, login and logout

# Description
Used identity scaffolding to provide the user with a registration, login and logout


## Related Ticket(s)
[ticket 1]https://github.com/sydneywait/AppalachianHarvest/issues/1


## Steps to Test
- Add and commit on your own branch before checking out to sw-identity.  Pull down the most recent version.
- Run the compiler and use the url https://localhost:5001/
- Register a new user entering first name, last name, email, and password, then add a profile picture
- you should be logged in after hitting submit.
- you can logout and log back in using the credentials for the user you just created.



# Checklist:
